### ISSUE TITLE
An error occurrs after operators like  `Conv2d`, `BatchNorm2d`, and `ReLU`  are fused

### ISSUE THREAD

**Issue by AnonymousChaseDreamer** [pytorch/pytorch#111322]
### 🐛 Describe the bug

Hello, when I use Pytorch to fuse the operators in an `original model` like below:
```
ModuleList(
  (0): Linear(in_features=1, out_features=258, bias=True)
  (1): Conv2d(4, 5, kernel_size=(2, 2), stride=(64, 64), dilation=(4, 2))
  (2): BatchNorm2d(5, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
  (3): ReLU()
)
```

I found that the `fused model` only contains `ConvReLU2d` operator. `BatchNorm2d` is missing like below:
```
ModuleList(
  (0): Linear(in_features=1, out_features=258, bias=True)
  (1): ConvReLU2d(
    (0): Conv2d(4, 5, kernel_size=(2, 2), stride=(64, 64), dilation=(4, 2))
    (1): ReLU()
  )
  (2): Identity()
  (3): Identity()
)
```

 However, in Pytorch document, these three operators can be fused as expected. It is not clear why this problem occurs.

Moreover, the output of the model should be consistent before and after operator fusion. But my fuzzer detected an inconsistency, which I suspect is due to a problem with the `pytorch operator fusion` itself. Here is the code to reproduce the inconsistent results:
```python
import torch
import torch.nn as nn
import torch.nn.functional as F
import json

checkpoint = torch.jit.load('./model.pt')
for name, param in checkpoint.named_parameters():
    print(f"Layer: {name}")
    print(f"Weight:\n{param.data}")
    print(f"Bias:\n{param.data}")
    print("\n")


input_file = "./torch_origin_model.input.0.json"  # the input of original model or fused model is the same

with open(input_file, 'r') as file:
    data = json.load(file)


input_list = []
for key, value in data.items():
    input_list.append(value)


tensor_data = torch.tensor(input_list[0], dtype=torch.float32)

checkpoint.eval()

with torch.no_grad():
    output = checkpoint(tensor_data)

print('---------')

print(output)
```

I have uploaded relevant files to this link ([https://drive.google.com/drive/folders/1N_NZ0-SdZKrE3hpVIhZSgI6mZHChmKhU?usp=sharing](https://drive.google.com/drive/folders/1N_NZ0-SdZKrE3hpVIhZSgI6mZHChmKhU?usp=sharing))

Thx in advance :)

### Versions

Collecting environment information...
PyTorch version: 1.13.1
Is debug build: False
CUDA used to build PyTorch: Could not collect
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.6 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: Could not collect
CMake version: version 3.8.2
Libc version: glibc-2.17

Python version: 3.7.16 (default, Jan 17 2023, 22:20:44)  [GCC 11.2.0] (64-bit runtime)
Python platform: Linux-4.15.0-213-generic-x86_64-with-debian-buster-sid
Is CUDA available: False
CUDA runtime version: Could not collect
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-32GB
GPU 1: Tesla V100-SXM2-32GB
GPU 2: Tesla V100-SXM2-32GB
GPU 3: Tesla V100-SXM2-32GB

Nvidia driver version: 470.182.03
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.21.6
[pip3] onnx==1.14.1
[pip3] onnxoptimizer==0.3.13
[pip3] onnxruntime==1.14.1
[pip3] torch==1.13.1
[pip3] torchaudio==0.13.1
[pip3] torchvision==0.14.1
[conda] Could not collect
```[tasklist]
### Tasks
```

cc @EikanWang @jgong5 @wenzhe-nrv @sanchitintel
